runtime/service/src/main/java/com/alipay/muagent/service/mybatisplus/dto/ToolConverter.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Alipay.com Inc.
+ * Copyright (c) 2004-2024 All Rights Reserved.
+ */
+package com.alipay.muagent.service.mybatisplus.dto;
+
+import com.alipay.muagent.model.tool.meta.Tool;
+import com.alipay.muagent.util.GsonUtils;
+
+/**
+ * @author chenjue.wwp
+ * @version $Id: ToolConverter.java, v 0.1 2024-11-22 14:43  Exp $$
+ */
+public class ToolConverter extends Converter<ToolDO, Tool> {
+
```

Comment:
it would be better to add a static build() method so that it can avoid to new a instance explicitly every time.